qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteDefinition.java

```diff
@@ -270,6 +272,74 @@ private CompletableFuture<List<? extends LocationLink>> findDefinitionFromPart(P
 		}
 	}
 
+	/**
+	 * Find parameter definition for Enum in #when section.
+	 *
+	 * @param parameter
+	 * @param template
+	 * @param cancelChecker
+	 *
+	 * @return parameter definition for Enum in #when section.
+	 */
+	private CompletableFuture<List<? extends LocationLink>> findDefinitionFromParameter(Parameter parameter,
+			Template template, CancelChecker cancelChecker) {
+		Section caseSection = parameter.getOwnerSection();
+		if (!isCaseSection(caseSection)) {
+			return NO_DEFINITION;
+		}
+		Section whenSection = caseSection.getParentSection();
+		if (!isWhenSection(whenSection)) {
+			return NO_DEFINITION;
+		}
+		if (whenSection.isValidOperator(parameter.getName()) && isFirstParameter(parameter, caseSection)) {

```

Comment:
I think it should benice to write

```java
if (caseSection.isOperator(parameter)) {
    return NO_DEFINITION;
} 
```

in other words create isOperator(Parameter) in new class BaseCaseSection.